validation_components/spreadsheet_parsing.py

```diff
@@ -28,16 +33,20 @@ def report_error(self, error_code, ncbi_common_name=None):
 
 class NcbiQuery:
 
+    @staticmethod
+    def get_now():
+        return datetime.now()
+
```

Comment:
I think there is indeed no better solution to this. See also discussion on https://stackoverflow.com/questions/4481954/trying-to-mock-datetime-date-today-but-not-working. So happy with this solution.